src/Text/Pandoc/App/Opt.hs

```diff
@@ -184,41 +184,41 @@ instance FromJSON Opt where
        <$> o .:? "tab-stop" .!= optTabStop defaultOpts
        <*> o .:? "preserve-tabs" .!= optPreserveTabs defaultOpts
        <*> o .:? "standalone" .!= optStandalone defaultOpts
-       <*> o .:? "from"
-       <*> o .:? "to"
+       <*> o .:? "from" .!= optFrom defaultOpts
+       <*> o .:? "to" .!= optTo defaultOpts
```

Comment:
optForm and optTo defaultOpts are both Nothing, so adding this has no effect. Perhaps that's why I left it off.
Similarly for optTemplate.
Haven't checked all the others.